Dell/recovery_common.py

```diff
@@ -29,7 +29,7 @@
 gi.require_version('UDisks', '2.0')
 from gi.repository import GLib, UDisks
 import os
-import shutil
+#import shutil
```

Comment:
Don't comment out an import, if it's not needed - please remove it.

Reply:
Thanks, will remove it.